chapters/fa/chapter2/2.mdx

```diff
@@ -318,7 +318,7 @@ outputs = model(inputs)
 {/if}
 
 
-حالا اگر نگاهی به ویژگی shape ورودی‌ها بیاندازیم، خوهیم دید که تعداد ابعاد بردارهای نماینده بسیار کمتر است: قسمت سر مدل بردارهای چندین بعدی که قبلا دیدیم را به عنوان ورودی دریافت کرده، و به عنوان خروجی بردارهایی با دو عضو(یکی برای هر دسته در عملیات دستبندی) تولید کرده است.
+اگر نگاهی به شکل ورودی‌ها بیاندازیم، خواهیم دید که حالا تعداد ابعاد آنها بسیار کمتر است: قسمت سَر مدل، بردارهای با بُعد بالا که قبلا دیدیم را به عنوان ورودی دریافت کرده و به عنوان خروجی بردارهایی با دو عضو(یکی برای هر دسته در عملیات دستبندی) تولید کرده است.
```

Comment:
Fixes:

- "دستبندی" -> "دسته‌بندی"
- there must be a space before parentheses openings

Suggestion:
قسمت سَر مدل، بردارهای بُعدبالایی را که قبلا دیدیم به عنوان ورودی دریافت کرده و در خروجی خود، بردارهایی با دو عضو (یکی به ازای هر برچسب دسته‌بندی) تولید می‌کند.
instead of:
قسمت سَر مدل، بردارهای با بُعد بالا که قبلا دیدیم را به عنوان ورودی دریافت کرده و به عنوان خروجی بردارهایی با دو عضو(یکی برای هر دسته در عملیات دستبندی) تولید کرده است.

Now, I know you're against "برچسب" translation for "label" term, but I'm afraid this is the closest equivalent we have so far. The other motive is that it has been widely used as the equivalent in Persian ML literature. The alternatives that come to my mind right now would be to transliterate here and use "لیبل", or use "گروه دسته‌بندی" instead.
I have to admit, using "دسته" as an equivalent for "label" is far from efficient.

Reply:
Applied all.

Went with:
قسمت سَر مدل، بردارهای بُعد بالایی که قبلا دیدیم را به عنوان ورودی دریافت کرده و در خروجی خود، بردارهایی با دو عضو (یکی به ازای هر برچسب دسته‌) تولید می‌کند.

True, not a fan of برچسب but have since warmed up to it and accepted it elsewhere, don't remember exactly where though, maybe in a review. This review has been open for too long and that's why there were some inconsistencies in these pages with G. All sorts of problems creep up when PRs are left unattended for too long.